SECURITY.md

```diff
@@ -0,0 +1,9 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+To report a security issue, please click on the "Report a vulnerability" button from the "Security" tab. 
+See detailed [instructions]([url](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability#privately-reporting-a-security-vulnerability)https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability#privately-reporting-a-security-vulnerability).
+
+The maintainers will respond within 3 working days of your email.
```

Comment:
Won't we respond on the issue? For email we'd need to have a PGP key published here...

Reply:
Oops. Bad copy-and-paste. Updated.